File: workspace/ASTIC-lib/src/ro/info/asticlib/query/Result.java
package ro.info.asticlib.query;

public class Result{
	private String filePath;
	private int weight;
	private boolean isPrimary;
	private String clusterName;
	
	public Result(String filePath, int weight, boolean isPrimary,
			String clusterName) {
		super();
		this.filePath = filePath;
		this.weight = weight;
		this.isPrimary = isPrimary;
		this.clusterName = clusterName;
	}
	public String getFilePath() {
		return filePath;
	}
	public void setFilePath(String filePath) {
		this.filePath = filePath;
	}
	public int getWeight() {
		return weight;
	}
	public void setWeight(int weight) {
		this.weight = weight;
	}
	public boolean isPrimary() {
		return isPrimary;
	}
	public void setPrimary(boolean isPrimary) {
		this.isPrimary = isPrimary;
	}
	public String getClusterName() {
		return clusterName;
	}
	public void setClusterName(String clusterName) {
		this.clusterName = clusterName;
	}
	
	
}

File: ASTIC_client_V2/ASTIC_client_V2/query/Result.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace ASTIC_client.query
{
    class Result
    {
        private String filePath;
        private float weight;
        private bool isPrimary;
        private String clusterName;

        public Result(String filePath, int weight, bool isPrimary,String clusterName)
        {
            this.filePath = filePath;
            this.weight = weight;
            this.isPrimary = isPrimary;
            this.clusterName = clusterName;
        }
        public String getFilePath()
        {
            return filePath;
        }
        public void setFilePath(String filePath)
        {
            this.filePath = filePath;
        }
        public float getWeight()
        {
            return weight;
        }
        public void setWeight(int weight)
        {
            this.weight = weight;
        }
        public bool getIsPrimary()
        {
            return isPrimary;
        }
        public void setPrimary(bool isPrimary)
        {
            this.isPrimary = isPrimary;
        }
        public String getClusterName()
        {
            return clusterName;
        }
        public void setClusterName(String clusterName)
        {
            this.clusterName = clusterName;
        }
    }
}
